### DISCUSSION TITLE
Specifying cache store?

### DISCUSSION THREAD

**Discussion by binaryfire** [worksome/exchange#13] [Q&A] [answered]
Great package guys! I’m building a multi tenant app which uses multiple cache stores. Is there a way to specify the store this package uses?

**Comment by olivernybroe** [comment, marked as answer]
I have created a PR which would add support for setting the store :) 
https://github.com/worksome/exchange/pull/14

**Reply by binaryfire** [reply]
Holy crap that was fast 😂 Thanks!
